pkg/agent/config/node_config.go

```diff
@@ -80,6 +80,10 @@ type WireGuardConfig struct {
 	MTU int
 }
 
+type EgressExceptConfig struct {
```

Comment:
ditto, we should reuse this struct for other egress related configurations in the future. Naming it `EgressConfig` would be more extensible.

Reply:
Agreed.